libs/blocks/global-navigation/global-navigation.js

```diff
@@ -536,8 +509,8 @@ class Gnav {
     };
 
     await Promise.all([
-      loadScript(`https://${environment}.adobeccstatic.com/unav/1.0/UniversalNav.js`),
-      loadStyle(`https://${environment}.adobeccstatic.com/unav/1.0/UniversalNav.css`),
+      loadScript(`https://${environment}.adobeccstatic.com/unav/1.1/UniversalNav.js`),
```

Comment:
I know this is a feature branch, but if we're planning to get this to STAGE soon, then we need to include @salonijain3 and her team for awareness, they have a similar request to update the universal navigation version.

Reply:
Yes, this definitely needs to be considered before merging to `stage`, I've asked the UNav team when we can use this new version on production, but haven't seen a reply yet.